egs2/bibletts/tts1/conf/tuning/finetune_joint_conformer_fastspeech2_hifigan.yaml

```diff
@@ -0,0 +1,271 @@
+# This EXPERIMENTAL configuration is for ESPnet2 to finetune
```

Comment:
We would like to minimize the config size as much as possible. For new recipes, please try to just upload the one with regard to your pre-trained models.